src/ts/ConvivaAnalytics.ts

```diff
@@ -516,7 +548,7 @@ export class ConvivaAnalytics {
   private onVideoQualityChanged = (event: VideoQualityChangedEvent) => {
     // We calculate the bitrate with a divisor of 1000 so the values look nicer
     // Example: 250000 / 1000 => 250 kbps (250000 / 1024 => 244kbps)
```

Comment:
? What is it now? `/1000` or `/1024`
Also I think the comment is no longer needed, as this info is present in the method name of the `BitrateHelper`

Reply:
it's `/ 1000` but you are right its obsolete and also duplicated in the `BitrateHelper`